ci-operator/step-registry/baremetalds/e2e/ovn/bgp/pre/baremetalds-e2e-ovn-bgp-pre-commands.sh

```diff
@@ -83,11 +93,43 @@ deploy_frr_external_container() {
   cp "${FRR_TMP_DIR}"/frr-k8s/hack/demo/frr/daemons $FRR_CONFIG
   chmod a+rw $FRR_CONFIG/*
 
-  podman rm -f frr
-  podman run -d --privileged --network host --rm --ulimit core=-1 --name frr --volume "$FRR_CONFIG":/etc/frr quay.io/frrouting/frr:9.1.0
+  podman run -d --privileged --network agnhost_net --ip 172.20.0.2 --rm --ulimit core=-1 --name frr --volume "$FRR_CONFIG":/etc/frr quay.io/frrouting/frr:9.1.0
+  podman network connect node_net frr
+  # assign static IP to the interface that connects to the node network
+  podman exec frr ip addr add dev eth1  192.168.111.3/24
+  podman exec frr ip -6 addr add dev eth1  fd2e:6f44:5dd8:c956::3/64
+  # remove the default route given by podman, use node network gateway as the default route
```

Comment:
wouldn't it be better to connect `frr` to `node_net` on `podman run` and then connect it to `agnhost_net`  and just have to assign the IPs?

Reply:
This node_net is created with a libvrit managed bridge. So `podman run` cannot set IP address for the container. We have to do it in this way.

Reply:
actually, I am a bit confused, give me a min